### DISCUSSION TITLE
Different approach to self-update

### DISCUSSION THREAD

**Discussion by ap1969** [wailsapp/wails#2720] [Ideas]
Hi, looking to see if this is feasible.

I use capacitorjs for mobile apps, which again is a binary with webview.

It's possible to do frontend updates to capacitor by

A. Downloading a new zip of the Web app
B. Extracting that to a folder
C. Pointing webview to that folder

Is something similar possible with Wails?

Regards,
Andy

**Comment by mooijtech** [comment]
References https://github.com/minio/selfupdate

**Comment by leaanthony** [comment]
> Hi, looking to see if this is feasible.
> 
> I use capacitorjs for mobile apps, which again is a binary with webview.
> 
> It's possible to do frontend updates to capacitor by
> 
> A. Downloading a new zip of the Web app B. Extracting that to a folder C. Pointing webview to that folder
> 
> Is something similar possible with Wails?
> 
> Regards, Andy

Not unless you want to ship your application assets outside of the binary which is not how Wails works (currently).
